com.blitterstudio.amiberry.yml

```
@@ -0,0 +1,116 @@
+app-id: com.blitterstudio.amiberry
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: amiberry.sh
+rename-icon: amiberry
+copy-icon: true
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  # Wayland access
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Needs to talk to the network:
+  - --share=network
+  # Allow bluetooth (e.g. controllers)
+  - --allow=bluetooth
+  # Play sounds using PulseAudio
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  
+modules:
+  - name: libmpeg2
+    config-opts:
+      - --disable-static
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
```

```suggestion
      - "*.la"
      - "*.a"
```
never there is .la without .a